src/test/java/plugins/AnalysisCollectorPluginTest.java

```diff
@@ -329,6 +330,36 @@ public void should_aggregate_warnings_in_dashboard_portlet() {
         assertThat(dashboard, not(hasWarningsFor(job, TASKS, TASKS_ALL)));
     }
 
+    @Test
+    @WithPlugins({"git@2.4.0", "workflow-aggregator@1.10"})
+    public void should_compute_annotations_on_workflow() {
+        MavenInstallation.installMaven(jenkins, "M3", "3.3.3");
+        WorkflowJob job = jenkins.jobs.create(WorkflowJob.class);
+        // TODO: Job.copyResource(String) does not work for WorkflowJob, since it does not owns
+        // a workspace but the build does (actually it could own 0..N workspaces).
+        job.script.set(
+            "node {\n" +
+            "  git 'https://github.com/amuniz/maven-helloworld.git'\n" +
```

Comment:
Agree. I was thinking about it (thus the TODO in line 338). I can implement the proposal (described in [this  comment](https://github.com/jenkinsci/acceptance-test-harness/pull/40/files#r40503584)) as part of this PR if there is general agreement on it.

It would avoid the use of a external repository.

BTW we should fix this in [other places](https://github.com/jenkinsci/acceptance-test-harness/blob/master/src/test/java/plugins/WorkflowPluginTest.java#L72).